doc/docs/master/how-tos/developer-workflow/working-with-pipelines.md

```diff
@@ -110,32 +114,14 @@ create a repository for your project.
      docker push <image>:tag
      ```
 
-For more information about building Docker images, see
-[Docker documentation](https://docs.docker.com/engine/tutorials/dockerimages/).
-
-## Step 3: Load Your Data to Pachyderm
-
-You need to add your data to Pachyderm so that your pipeline runs your code
-against it. You can do so by using one of the following methods:
-
-* By using the `pachctl put file` command
-* By using a special type of pipeline, such as a spout or cron
-* By using one of the Pachyderm's [language clients](../reference/clients/)
-* By using a compatible S3 client
-* By using the Pachyderm UI (Enterprise version or free trial)
-
-For more information, see [Load Your Data Into Pachyderm](../load-data-into-pachyderm/).
+!!! note
+    TODO Pipelines require a unique tag
```

Comment:
Just a note to address this todo before merging

Reply:
Thanks for catching this! Will address.